### PR TITLE
Show the hidden eyeball in submission list

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-12986
affects: Teacher
release note: none

Pork: Update small submission list design changes
(remove dosclosure indicator, add double hyphen for
no grade)

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1004968&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: rn/Teacher/src/images/index.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ const images = {
   relax: require('./relax.png'),
   hamburger: require('./hamburger.png'),
   invisible: require('./invisible.png'),
+  off: require('./off.png')
```
Comment (bluwave) [inline rn/Teacher/src/images/index.js]: is this the eyeball?

Comment (shichongrui) [reply rn/Teacher/src/images/index.js]: It is the eyeball with a line through it.

Comment (shichongrui) [reply rn/Teacher/src/images/index.js]: image

Comment (bluwave) [reply rn/Teacher/src/images/index.js]: ah, we don't have that one in inst-icons do we?  👍

Comment (shichongrui) [reply rn/Teacher/src/images/index.js]: Unfortunately no. I thought of making a component that was the existing eyeball and a line but then decided to just add the icon and keep things simpler.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Teacher

[MBL-12986](https://instructure.atlassian.net/browse/MBL-12986)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 94.9% | 94.87% | 0.03%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**